meeting-tsc/2023-12-06-tsc.md

```diff
@@ -3,9 +3,51 @@
 Date: Wednesday, December 6, 2023 - 9AM/12PM PT/ET | [Google Meet Link](https://meet.google.com/tes-qcdo-jpf)
 
 Participants:
+* Ann Ming Samborski (BastionZero)
+* Ethan Heilman (Bastionzero)
+* Joel Kamp (Docker)
+* Jonny Stoten (Docker)
+* Lucie Mugnier (BastionZero)
 
 # Agenda
+1. [TSC chair election](#tsc-chair-election)
+2. [Discuss how we document and standardize OpenPubkey](#discuss-how-we-document-and-standardize-openpubkey)
+    - What format do we use? RFC-like Mediawiki?
+    - Where do we store the standards?
+    - How do we propose, review and approve /reject it?
+3. [Ownership of non-core OpenPubkey repositories](#ownership-of-non-core-openpubkey-repositories)
+    - [Verify-docker-cli-plugin](https://github.com/openpubkey/verify-docker-cli-plugin)
+    - [Signed-attestation](https://github.com/openpubkey/signed-attestation)
+    - [Buildkit](https://github.com/openpubkey/buildkit)
+    - [Demo](https://github.com/openpubkey/demo)
+4. [Standardizing admin/write access to repositories](#standardizing-adminwrite-access-to-repositories)
 
 # Meeting Notes
 
-# Action Items
\ No newline at end of file
+## TSC Chair Election
+
+Ethan is chair by unanimous agreement by TSC members. 
+
+## Discuss How We Document and Standardize OpenPubkey
+
+RFC is a nice format, but there are concerns it is too formal to serve the community as a whole. 
+
+The goal is to give the community a structured way to interact with the design of OPK rather than just the code. 
+
+For now, we think a Markdown file in GitHub makes sense (within the `openpubkey/openpubkey` repo) so folks can make PRs against it. We would like to adopt the RFC that defines usage of "must," "should," and "may." Diagrams in Markdown can use Mermaid or if a more complex diagram is necessary, can always embed an image. 
```

Comment:
I mentioned https://github.com/in-toto/attestation/tree/main/spec/v1 and https://github.com/secure-systems-lab/dsse/blob/master/protocol.md as nice examples of specs in markdown on Github.

Reply:
DSSE Protocol looks really nice. It would be a nice example to follow